src/AggregateSpecsPlugin.ts

```diff
@@ -14,10 +14,48 @@ const DOUBLE_PRECISION_OID = "701";
 const INTERVAL_OID = "1186";
 const MONEY_OID = "790";
 
+const NUMERIC_CATEGORY = "N";
+const DATE_TIME_CATEGORY = "D";
+const NETWORK_ADDRESS_CATEGORY = "I";
+const STRING_CATEGORY = "S";
+
+const ENUM_TYPE = "p";
+
+const SUM_DEFAULTS = Object.freeze({
+  [MONEY_OID]: "0::money",
+  [INTERVAL_OID]: "INTERVAL '0 DAY'",
```

Comment:
These need to be SQL fragments to allow this:
```suggestion
  [MONEY_OID]: sql.fragment`0::money`,
  [INTERVAL_OID]: sql.fragment`INTERVAL '0 DAY'`,
```

With that in mind, they'll also need to be defined in a scope that has access to `const sql = build.pgSql`